packages/devtools_app/lib/src/debugger/program_explorer_controller.dart

```diff
@@ -37,6 +37,8 @@ class ProgramExplorerController extends DisposableController
   /// The currently selected node.
   VMServiceObjectNode _selected;
 
+  VMServiceObjectNode _outlineSelection;
```

Comment:
add dartdoc to this member and modify the dart doc (and possibly name) of `_selected` above to differentiate between the two